app/frontend/src/cljs/teet/contract/contract_partners_view.cljs

```diff
@@ -241,12 +245,129 @@
+                                  :class "submit"
+                                  :on-click validate}
+          (tr [:buttons :save])])]]]))
+
+(defn add-personnel-form
+  [e! {:keys [query] :as app} selected-partner]
+  (r/with-let [form-atom (r/atom {})]
+    (let [selected-user (:company-contract-employee/user @form-atom)
+          user-selected? (boolean
+                           selected-user)]
+      [Grid {:container true}
+       [Grid {:item true
+              :xs 12
+              :md 6}
+        [form/form2 {:e! e!
+                     :autocomplete-off? true
+                     :value @form-atom
+                     :on-change-event (form/update-atom-event form-atom (fn [old new]
+                                                                          (if (= {:company-contract-employee/role #{}} new)
+                                                                            (dissoc old :company-contract-employee/role)
+                                                                            (merge old new))))
+                     :spec :thk.contract/add-contract-employee
+                     :cancel-event #(common-controller/map->NavigateWithExistingAsDefault
+                                      {:query (merge query
+                                                     {:page :partner-info})})
+                     :save-event #(common-controller/->SaveFormWithConfirmation
+                                    :thk.contract/add-contract-employee
+                                    {:form-value @form-atom
+                                     :company-contract-eid (:db/id selected-partner)}
+                                    (fn [_response]
+                                      (fn [e!]
+                                        (e! (common-controller/->Refresh))
+                                        (e! (common-controller/map->NavigateWithExistingAsDefault
+                                              {:query (merge
+                                                        query
+                                                        {:page :partner-info})}))))
+                                    (tr [:contract :employee-added]))}
+         [typography/Heading1 {:class (<class common-styles/margin-bottom 1.5)}
+          (tr [:contract :add-person])]
+         (if selected-user
+           [selected-user-information selected-user]
+           [form/field :company-contract-employee/user
+            [select/select-search
+             {:e! e!
+              :query (fn [text]
+                       {:args {:search text
+                               :company-contract-id (:db/id selected-partner)}
+                        :query :contract/possible-partner-employees})
+              :format-result select/user-search-select-result
+              :after-results-action {:title (tr [:contract :add-person-not-in-teet])
+                                     :on-click #(println "This is placeholder")
```

Comment:
what should it do? remove placeholder

Reply:
It's for the next issue to create a form to add totally new users to teet. But you're right, removed it from this PR.